src/analysis/normed/group/basic.lean

```diff
@@ -172,6 +172,9 @@ lemma norm_add_le_of_le {g₁ g₂ : E} {n₁ n₂ : ℝ} (H₁ : ∥g₁∥ ≤
   ∥g₁ + g₂∥ ≤ n₁ + n₂ :=
 le_trans (norm_add_le g₁ g₂) (add_le_add H₁ H₂)
 
+lemma norm_add₃_le (x y z : E) : ∥x + y + z∥ ≤ ∥x∥ + ∥y∥ + ∥z∥ :=
```

Comment:
Cf. `abs_add_three`.

Reply:
Do you want to renamed this lemma? I think `abs_add_three` is a bad name because it looks like it's about `|a + 3|`.